src/main/java/seedu/linkedout/model/applicant/NameContainsAllKeywordsPredicate.java

```diff
@@ -28,4 +28,9 @@ public boolean equals(Object other) {
                 && keywords.equalsIgnoreCase(((NameContainsAllKeywordsPredicate) other).keywords)); // state check
     }
 
+    @Override
+    public int numberOfKeywordMatches(Applicant applicant) {
+        return 1;
+    }
```

Comment:
Could you explain what this function is for in this context of NameContainsAllKeywordsPredicate, thank you😅

Reply:
NameContainsAllKeywordsPredicate is only used by view and since all the NameContainsKeywordPredicates, JobContainsKeywordPredicates and NameContainsAllKeywordsPredicate extends from abstract Keyword Predicate class, I will need to override it (although it seems a bit redundant). View can only view 1 applicant at one time so it only return 1. Is it better if I write javadoc comment even for overriding methods?